packages/flame/lib/src/experimental/column_component.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flame/src/experimental/layout_component.dart';
+import 'package:flutter/widgets.dart';
+import 'package:vector_math/vector_math_64.dart';
+
+/// Allows laying out children in a column by defining a [MainAxisAlignment]
+/// type.
+class ColumnComponent extends LayoutComponent {
+  ColumnComponent({
```

Comment:
It would be good to add docs here, as it is not clear what the parameters mean. Especially `size` is the tricky one here...